tests/pip/reqs-included.txt

```diff
@@ -0,0 +1,3 @@
+-r temp1.txt
+--requirement=temp2.txt
+nltk==3.2.1
```

Comment:
Wow! nltk is not the smallest package for the test. I think you should pick the package that already used in some container, just put it in another requirements file and require it from there. You don't even need to install two packages. The upside is that package is already cached in pip cache, so the test will be fast.